src/styles/typography/link-new-tab/index.njk

```diff
@@ -0,0 +1,6 @@
+---
+title: Links in new tabs  – Typography
+layout: layout-example.njk
+---
+
+<a href="#" class="govuk-link" target="_blank">govuk-link (opens in new tab)</a>
```

Comment:
@StephenGill As you already suggested could we also add 
```
rel="noreferrer noopener"
```
to reduce the risk of [reverse tabnabbing](https://owasp.org/www-community/attacks/Reverse_Tabnabbing).

Sources:
- [HTML spec link type noreferrer](https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer)
- [Target="_blank" - the most underestimated vulnerability ever](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)